.gitignore

```diff
@@ -6,7 +6,7 @@ yarn-debug.log*
 yarn-error.log*
 pnpm-debug.log*
 lerna-debug.log*
-
+coverage/
```

Comment:
Please remove this. If you rebased correctly, you will notice that it was already added in a previous P.R